test/cutting/cut_finding/test_cut_finder_results.py

```diff
@@ -190,7 +190,7 @@ def test_four_qubit_circuit_two_qubit_qpu(
     )  # circuit separated into 2 subcircuits.
 
     assert (
-        optimization_pass.get_stats()["CutOptimization"] == array([15, 46, 15, 6])
+        optimization_pass.get_stats()["CutOptimization"] == array([11, 36, 15, 4])
```

Comment:
Is there some sense I can make of these indices? (I think they're indices). How do you know what they should be here?

Reply:
They're basically just stats that keep track of things like how many states were added to the queue and how many backjumps were performed during the search. These numbers were obtained just by running the search algorithm on these circuits.

Reply:
OK it's dubious to let the output of your function determine the target values in your unit tests. It's better to manually track down whether these values are correct when making the test case. I understand the temptation in situations like these, but doing this can give you a false sense of security and defeat the purpose of unit testing.

Reply:
I think it was partly also because I was not able to come up with a more creative test for this function.

Reply:
I would probably remove the test if it were my code, since there could very well be a bug here that is treated as ground truth just because that was the output.

It's kind of outside the scope of this PR, but I just wanted to give my $.02 on that :D.